src/components/provider/PyatchProvider.jsx

```diff
@@ -377,7 +377,9 @@ const PyatchProvider = props => {
       handleSaveTargetThreads(pyatchVM.editingTarget);
     }
     await pyatchVM.deleteSprite(targetId);
+    const index = targetIds.indexOf(targetId);
     setTargetIds(targetIds.filter(id => id !== targetId));
+    pyatchVM.setEditingTarget(targetIds[index > 1 ? index - 1 : 0]);
```

Comment:
Can you refactor that index into a separate variable with a descriptive name? Just to make the code a little more readable.